samples/msal-node-samples/custom-INetworkModule-and-network-tracing/HttpClientCurrent.ts

```diff
@@ -287,7 +296,7 @@ const networkRequestViaHttps = <T>(
     }
 
     return new Promise<NetworkResponse<T>>((resolve, reject) => {
-        const request = https.request(url, customOptions);
+        const request = https.request(customOptions);
```

Comment:
Can you please create a minimal repro and open a bug on NodeJS repo and see what they say ? It looks to me that the issue is with them.

https://github.com/nodejs/node/blob/v13.14.0/lib/_http_client.js#L86

Reply:
Yes, this issue is definitely on Node.js. I tested it the other day and had the same issue.

Reply:
@bgavrilMS I'm unable to create a minimal repro that reproduces the bug.